manifests/k/KiCad/KiCad/Lite/5.1.12_1/KiCad.KiCad.Lite.locale.en-US.yaml

```
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: KiCad.KiCad.Lite
+PackageVersion: 5.1.12_1
+PackageLocale: en-US
+Publisher: KiCad
+PublisherSupportUrl: https://www.kicad.org/help/
+Author: KiCad
+PackageName: KiCad

```

```suggestion
PackageName: KiCad Lite
```
To make sure it doesn't conflict with the non-Lite version.

---

Thanks for the correction was a copy&paste error